src/helpers/helpers.ts

```diff
@@ -17,11 +17,17 @@ tmp.setGracefulCleanup(); // cleanup temp dirs even when an uncaught exception o
 const now = new Date();
 const TMP_TIME = `${now.getHours()}-${now.getMinutes()}-${now.getSeconds()}`;
 
-type DownloadResult = {
+export type DownloadResult = {
   data: Buffer;
   ext: string;
 };
 
+type ProcessEnvs = Record<string, unknown>;
```

Comment:
Is this better than `{ [key: string]: unknown }` ?

Reply:
I don't think so. Just different ways of saying the same thing.